roles/manager/templates/wrapper/osism-update-manager.j2

```diff
@@ -22,7 +22,7 @@ if [[ ! -x "$(command -v ansible-playbook)" ]]; then
 
         virtualenv -p python3 $VENV_PATH
         source $VENV_PATH/bin/activate
-        pip3 install --no-cache-dir "$ANSIBLE_VERSION"
+        pip3 install --no-cache-dir "$ANSIBLE_VERSION" netaddr
```

Comment:
Use ``netaddr==0.8.0``

Reply:
used NETADDR_VERSION for this, is this OK?